launchflow/cli/main.py

```diff
@@ -570,20 +549,21 @@ async def import_resource_cmd(
             prompt_for_creation=True,
         )
     _set_global_project_and_environment(None, environment)
-    resource_refs = find_launchflow_resources(
-        scan_directory, ignore_roots=config.ignore_roots
-    )
-    service_refs = find_launchflow_services(
-        scan_directory, ignore_roots=config.ignore_roots
-    )
+    if resource is None:
```

Comment:
Is this overwriting your pr from earlier?

Reply:
oh yea it is.. good catch will fix